src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.spec.ts

```diff
@@ -17,6 +21,10 @@ describe('CopyQuestionsFromOtherSessionsModalComponent', () => {
         CommonModule,
         FormsModule,
         TeammatesCommonModule,
+        PanelChevronModule,
+        LoadingSpinnerModule,
+        LoadingRetryModule,
+        HttpClientTestingModule,
       ],
       providers: [
         NgbActiveModal,
```

Comment:
I think we should add more unit tests here

Reply:
Component tests have been added!